docs/spec-files/encryption.md

```diff
@@ -101,7 +101,7 @@ Assuming she also has Bob's DID document which happens to include the following
 }
 ```
 
-There should be only 1 entry in the recipients of the envelope, representing Bob. The corresponding `kid` header for this recipient MUST have `did:example:jklmnopqrstuvwxyz1#key-1` as value. The packing process MUST extract the public key bytes found in `publicKeyBase58` of Bob's DID Doc `KeyAgreement[0]` to execute the ECDH-1PU key derivation for content key wrapping.
+The recipients of the envelope must include all the KeyAgreement entries, representing Bob. The corresponding `kid` header for this recipient MUST have `did:example:jklmnopqrstuvwxyz1#key-1` as value as per the example above. The packing process MUST extract the public key bytes found in `publicKeyBase58` of Bob's DID Doc `KeyAgreement[0]` to execute the ECDH-1PU key derivation for content key wrapping.
```

Comment:
```suggestion
The recipients of the envelope MUST include all the KeyAgreement entries, representing Bob. The corresponding `kid` header for this recipient MUST have `did:example:jklmnopqrstuvwxyz1#key-1` as value as per the example above. The packing process MUST extract the public key bytes found in `publicKeyBase58` of Bob's DID Doc `KeyAgreement[0]` to execute the ECDH-1PU key derivation for content key wrapping.
```

Reply:
updated with suggested test @kdenhartog

Reply:
i will push a separate PR for this change.. for now, i'm reverting the update in this pr

Reply:
new PR to update this sentence: https://github.com/decentralized-identity/didcomm-messaging/pull/270